projects/plugins/jetpack/_inc/client/discussion/subscriptions.jsx

```diff
@@ -134,6 +139,17 @@ class SubscriptionsComponent extends React.Component {
 									'jetpack'
 								) }
 							/>
+							<ToggleControl
+								checked={ isSubscriptionsActive && this.props.getOptionValue( 'sm_enabled' ) }
+								disabled={
+									! isSubscriptionsActive ||
+									unavailableInOfflineMode ||
+									this.props.isSavingAnyOption( [ 'subscriptions' ] )
+								}
+								toggling={ this.props.isSavingAnyOption( [ 'sm_enabled' ] ) }
+								onChange={ this.handleSubscribeModalToggleChange }
+								label={ __( 'Enable the popup Subscribe Modal', 'jetpack' ) }
+							/>
```

Comment:
Can we have a link to edit the modal in the focused template part editor somewhere near this toggle? Ok in follow-up PR as well.

Reply:
Yup, I will handle this in a follow up PR.